htdocs/ticket/class/ticket.class.php

```diff
@@ -568,6 +567,13 @@ public function create($user, $notrigger = 0)
 				$this->id = $this->db->last_insert_id(MAIN_DB_PREFIX."ticket");
 			}
 
+			// Assign user if defined
```

Comment:
The create method is a low level CRUD method, so we should not have other business event and trigger inside the create.
Call of the assignUser should be done after calling the create by the parent page orchestrating the create call.

Reply:
Thanks for the feedback. I didn't know what was the best way to do this.

I'll make the required changes.